reference/entry.json

```
@@ -2353,6 +2353,84 @@
         ]
       }
     },
+    "/organizers/{organizerId}/url": {
+      "parameters": [
+        {
+          "$ref": "#/components/parameters/organizerId"
+        }
+      ],
+      "put": {
+        "summary": "Update Url",
+        "description": "Updates the url of the organizer with the given `organizerId`. The provided url can not be used for another organizer.",
```

```suggestion
        "description": "Updates the url of the organizer with the given `organizerId`. The provided url can not be in use on another organizer.",
```

(Otherwise it can be interpreted like it can only be used once for one organizer, and even if that organizer gets another URL later it cannot be used again)